security/fma-operator-fee-fix.md

```diff
@@ -0,0 +1,124 @@
+- **Risk Assessment**: Medium impact, high likelihood
+- **Mitigation:**
+    - Ensure acceptance tests assert no chain fork.
+
+### FM4: Generic items we need to take into account: `L1Block` badly hydrated
+
+- **Description:** At each hardfork, new data can be add to the `L1Block` contract, and the method called to hydrate it change (for instance
+  `setL1BlockValuesIsthmus` to `setL1BlockValuesJovian`). If there is a bug in a future method ending up to operator fee params no
+  longer being updated in the `L1Block` contract, the operator fee will no longer be taken into account in transactions fee.
+- **Risk Assessment:** medium severity / low likelihood
+- **Mitigations:**
+  The [Operator Fee Constistency](https://github.com/ethereum-optimism/optimism/blob/develop/op-e2e/actions/proofs/operator_fee_test.go) action test runs with all known hardforks activated at genesis, and checks that operator fee parameters are correctly reported to the `L1Block` contract. This has been updated to cover the new formula under Jovian.
+- **Detection:**
+  The action or E2E tests or local testing may pick up an issue.
+- **Recovery Path(s):**
+  - If the bug is located in op-node, a new version must be deployed.
+  - If the bug is located in the `L1Block` contract, the contract must be upgraded to fix the bug.
+
+### FM5: Overflow/Underflow of operator fee calculation
+
+- **Description:** If the operator fee calculation is vulnerable to overflows or underflows then it could be possible to manipulate the calculation through, say, a deliberately large gasUsed in order to trigger that overflow or underflow and charge a negative fee.
+- **Risk Assessment:** high severity / low likelihood
+- **Mitigations:**
+
+  - The calculation cannot overlow or underflow. See context below:
+
+    > `gas` is a uint64, so `(gasUsed * operatorFeeScalar * 100) + operatorFeeConstant` can be at most `(u64.max * u32.max * 100) + u64.max ~= 7.924660923989131e+33`, which is an int of bit length 103 and fits comfortably within a uint256 variable allocation.
+
+- **Recovery Paths(s):**
+  - A bug within the OP EVM would critical and would require an emergency upgrade of the sequencer and bridge.
+
+## Action Items
+
+Below is what needs to be done before launch to reduce the chances of the above failure modes occurring, and to ensure they can be detected and recovered from:
+
+**Initial review**
+
+- [ ] (NON-BLOCKING): Coordinate with wallet providers to update their fee estimation logic (assignee: TBD).
```

Comment:
I think https://github.com/ethereum-optimism/design-docs/pull/349/files#r2433833672 would be a blocking action item to add here?

Reply:
[70d2467](https://github.com/ethereum-optimism/design-docs/pull/349/commits/70d246701c3e69c66941686a2a200018e1ace00c)